ci/build/build-pipeline.yaml

```diff
@@ -54,6 +54,14 @@ spec:
         repository: amazoncorretto
         tag: 11
 
+    slack-notification: &slack-notification
+      on_failure:
+        put: notify
+        params:
+          text: |
+            Proxy Node pipeline failed on the `$BUILD_PIPELINE_NAME.$BUILD_JOB_NAME` step.
+            https://ci.london.verify.govsvc.uk/teams/proxy-node-build/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME
+
```

Comment:
nit: I personally prefer to create the anchor where I first use it.

Reply:
My thinking was if the anchor is defined at the top of the yaml, its not ambiguous in the pipeline which of `&` or `*` is the anchor or alias 🤷‍♂